src/services/middlewareServices.js

```diff
@@ -52,6 +56,10 @@ const PROVISION_SERVICES = [
   DEFAULT_SERVICES.RHSSO,
   DEFAULT_SERVICES.USER_RHSSO
 ];
+if (window.OPENSHIFT_CONFIG && window.OPENSHIFT_CONFIG.optionalProvisionServices.length > 0) {
```

Comment:
it just appends the services into the existing array (well actually concats it) is there are optional services available